app/controllers/users.js

```diff
@@ -0,0 +1,31 @@
+const userModel = require('../models').user,
+  usersService = require('../services/users'),
+  logger = require('../logger'),
+  bcrypt = require('bcryptjs');
+
+const encryptPassword = password => bcrypt.hashSync(password, bcrypt.genSaltSync());
```

Comment:
Move the logic for `bcrypt` to a service

Reply:
What about using an utils for this? Maybe this can be used in the future for another passwords or for another things.